lib/features/building_detail_view/presentation/widgets/building_data_source_link.dart

```diff
@@ -0,0 +1,32 @@
+import "package:flutter/widgets.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../../theme/app_theme.dart";
+import "../../../../utils/context_extensions.dart";
+
+class BuildingDataSourceLink extends ConsumerWidget {
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Text.rich(
+      TextSpan(
+        text: "${context.localize.data_come_from_website}: ",
+        style: const TextStyle(
+          fontWeight: FontWeight.bold,
+        ),
+        children: [
+          TextSpan(
+            text: "www.przewodnik.pwr.edu.pl",
```

Comment:
This string you can add to urlConfig Class as const value

Reply:
@tomasz-trela 

Thank You! You are completely right but it is temporary hardcoded string. Once the API is connected and URL provided in config files it will be replaced as in sks_menu_data_source_link.dart